docs/support.md

```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
 * In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
```

Comment:
It seems this sentence is almost the same than previous the bullet list, maybe we should "group" this.

Reply:
If you are using Xen Orchestra via the source and not XOA, you can ask questions and try to recieve help a number of different ways:

* In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
* In our IRC - `#xen-orchestra` on `Freenode`

We encourage you to give back to the community by assisting other users via these two avenues as well.